packages/zipkin-transport-kafka/src/KafkaLogger.js

```diff
@@ -51,7 +51,7 @@ module.exports = class KafkaLogger {
       });
     };
     try {
-      const encodedSpan = THRIFT.encode(span);
+      const encodedSpan = this.encoder.encode(span);
```

Comment:
make sure the message is a list of spans. https://github.com/openzipkin/zipkin/tree/master/zipkin-collector/kafka#json

since this is json it might be possible to do a test that actually shows this is the case (harder to grok a literal test with thrift)